modules/osdk-installing-cli.adoc

```diff
@@ -85,8 +79,7 @@ $ curl -OJL https://github.com/operator-framework/operator-sdk/releases/download
 ----
 --
 
-.. Place the binary and corresponding ASC file into the same directory and run
-the following command to verify the binary:
+.. Place the binary and corresponding ASC file into the same directory and run the following command to verify the binary:
```

Comment:
I don't know if we differ, but at least in ISC:

> Periods with file name extensions
Include a period before a file name extension. Assume that the period in a file name extension, such as .exe, is pronounced as dot, and use the indefinite article a.

> Example
Save your notes in a `.txt` file.